roles/tendrl-ansible.tendrl-server/tasks/tendrl-monitoring-integration.yml

```diff
@@ -19,7 +19,7 @@
     - rpm-installation
 
 - name: Initialize graphite-db
-  command: /usr/lib/python2.7/site-packages/graphite/manage.py syncdb --noinput
+  shell: PYTHONPATH=$GRAPHITE_ROOT/webapp/ django-admin migrate --settings=graphite.settings --run-syncdb
```

Comment:
Where is the upstream build of graphite 1.1.4? I don't see it in https://copr.fedorainfracloud.org/coprs/tendrl/dependencies/packages/ and epel7 provides  0.9.16-2.el7

Reply:
@TimothyAsirJeyasing is working on it

Reply:
Based on https://graphite.readthedocs.io/en/1.1.4/config-database-setup.html

Reply:
@GowthamShanmugam where is the `$GRAPHITE_ROOT` variable defined?

Reply:
Actually we are not setting any $GRAPHITE_ROOT here, i read initialization command it is actually taking local_settings.py configuration file form local_settings.py -> /etc/graphite-web/local_settings.py. i confirmed it is actaully reading this configuration file.

We don't need to give any PYTHONPATH  here, It contains nothing. So i am going to remove this.

Reply:
Actauly here pythonpath is emty only, it contains nothing, we are not export anythin to set environment variable. so we don't need to sepcify it.

Reply:
@GowthamShanmugam Ok, let's remove that. Then we can merge this.